flink-python/pyflink/table/table_result.py

```diff
@@ -75,9 +75,9 @@ def wait(self, timeout_ms: int = None):
         else:
             get_method(self._j_table_result, "await")()
 
-    def get_table_schema(self) -> TableSchema:
+    def get_resolved_schema(self) -> ResolvedSchema:
```

Comment:
we should not replace method but keep them around for 1-2 Flink versions. We should offer both and deprecate the old one.